drivers/adc/adc_stm32.c

```diff
@@ -1145,8 +1155,10 @@ static const struct adc_stm32_cfg adc_stm32_cfg_##index = {		\
 		.bus = DT_INST_CLOCKS_CELL(index, bus),			\
 	},								\
 	.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(index),			\
-	.has_temp_channel = DT_INST_PROP(index, has_temp_channel),		\
-	.has_vref_channel = DT_INST_PROP(index, has_vref_channel),		\
+	.vref_mv = DT_INST_PROP(index, vref_mv),			\
```

Comment:
Sorry, I have read it to fast. You are not adding a new property, but here you adding another way to get it the ADC reference value, which was already available from the standard API.